magmartfront/src/APIs/deletePokemon.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+function DeletePokemon( id: string ) {
+    return Promise.resolve(axios.delete(`http://localhost:3001/pokemon/${id}/del`))
```

Comment:
Padronizar com variável de ambiente.
```suggestion
    return Promise.resolve(axios.delete(`${process.env.API_URL}/${id}/del`))
```